src/internal/constants.ts

```diff
@@ -6,6 +6,41 @@ export const version = '1.9.0';
  */
 export const defaultSDKVersionForCompatCheck = '1.6.0';
 
+/**
+ * Minimum required client supported version for {@link getUserJoinedTeams} to be supported on {@link HostClientType.android}
+ */
+export const getUserJoinedTeamsSupportedAndroidClientVersion = '2.0.1';
+
+/**
+ * This is the SDK version when location APIs (getLocation and showLocation) are supported.
+ */
+export const locationAPIsRequiredVersion = '1.9.0';
+
+/**
+ * This is the SDK version when people picker API is supported on mobile.
+ */
+export const peoplePickerRequiredVersion = '2.0.0';
+
+/**
+ * This is the SDK version when captureImage API is supported on mobile.
+ */
+export const captureImageMobileSupportVersion = '1.7.0';
+
+/**
+ * This is the SDK version when media APIs is supported on all three platforms ios, android and web.
```

Comment:
There are 3 media APIs so please change is to are.

Reply:
changed.